### ISSUE TITLE
Automatically assign updated GE version

### ISSUE THREAD

**Issue by nning** [nning/protonutils#5]
See https://www.reddit.com/r/SteamPlay/comments/r49zfn/comment/hmjkx1d/?context=3

**Comment by crlmahlberg** [comment]
Indeed, an awesome idea! I cannot wait to batch update the GE version on multiple games. In fact, having to change more than a hundred of them one by one in Steam itself is what keeps me from using GE more at this point. **Hope to see the feature soon!**

**Comment by nning** [comment]
Thanks for your feedback! Auto-assign has some open questions but batch update looks like something straight forward to implement and I will probably have a look into this in the next weeks.

**Comment by crlmahlberg** [comment]
Thanks, can't wait for the update! By the way, did you ever think about a GUI version of your app? Would be even more useful, at least for me!

**Comment by nning** [comment]
Yes, a GUI version is on my list! It is not particularly valuable for me, though, as long as I do not have a SteamDeck. I would aim for easy GUI integration on SteamOS.

**Comment by crlmahlberg** [comment]
Great! One way to publish such a GUI version could be through [Flatpak](https://www.flatpak.org), which will be supported on both a Linux desktop and on the Steam Deck.

**Comment by nning** [comment]
Great idea! I created #18 to track this.

**Comment by crlmahlberg** [comment]
Glad I could help. Good luck!

**Comment by nning** [comment]
I created an experimental `compattool` command allowing for switching over all games from version A to B, for example:

    $ protonutils compattool migrate Proton-6.21-GE-2 Proton-7.0rc2-GE-1

Version 1.4.0 of `protonutils` contains that feature, feel free to test and give your feedback!
https://github.com/nning/protonutils/releases/tag/v1.4.0

**Comment by nning** [comment]
@crlmahlberg Thank you very much for reporting! I will have a look.
